### PR TITLE
Add Form Fields/Form Data to separate PR for size constraints

### PR DESCRIPTION
**Pull request by yakschuss**
## Description of change
This PR commits the lengthy pieces of the HLR refactor separately in order to get around the 'too large PR' issue from dangerbot on PR [5723](https://github.com/department-of-veterans-affairs/vets-api/pull/5723).

## Original issue(s)
[4527](https://vajira.max.gov/browse/API-4527)

This PR is a blocker for PR 5723

### CODE REVIEW

File: modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v1/form_fields.rb [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+module AppealsApi
+  module PdfConstruction
+    module HigherLevelReview::V1
+      class FormFields
+        def first_name
+          'F[0].#subform[2].VeteransFirstName[0]'
+        end
+
+        def middle_initial
+          'F[0].#subform[2].VeteransMiddleInitial1[0]'
```
Comment (LindseySaari) [inline modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v1/form_fields.rb:12]: You could probably pull out F[0].#subform[2] to a separate method to dry up the code a bit

File: modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v1/form_data.rb [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        def claimant_type(index)
+          index == 4 ? 1 : 'off'
+        end
+
+        def mailing_address_street
+          'USE ADDRESS ON FILE'
+        end
+
+        def mailing_address_unit_number
+          ''
```
Comment (LindseySaari) [inline modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v1/form_data.rb:56]: Are these methods meant to return empty strings?

Comment (yakschuss) [reply modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v1/form_data.rb:56]: Yep! The purpose of these classes was to abstract away the 'magic responses' that existed for the Higher Level Review PDF form data, and all the mailing address form fields were empty except the street address.

That value is 'USE ADDRESS ON FILE', so once that's updated, we'll have the correct values for the Veteran address.

File: modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v1/form_fields.rb (hunk 1 above)
Comment (yakschuss) [reply modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v1/form_fields.rb:12]: Good point! I'm going to do that.

Comment (LindseySaari) [reply modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v1/form_fields.rb:12]: Oh and pulling out F[0].#subform[3] would be another refactor too 👍

**Comment by LindseySaari** [comment]
It looks like dangerbot is erroring for too many LOC now. Could the form_data and form_fields changes be broken into separate PRs?

**Review by ElleeB — APPROVED** [review]
Great!

**Comment by yakschuss** [comment]
> It looks like dangerbot is erroring for too many LOC now. Could the form_data and form_fields changes be broken into separate PRs?

In theory, yes - although this PR is already a secondary PR to avoid the same issue! I'll figure this out 😂

**Comment by LindseySaari** [comment]
> > It looks like dangerbot is erroring for too many LOC now. Could the form_data and form_fields changes be broken into separate PRs?
> 
> In theory, yes - although this PR is already a secondary PR to avoid the same issue! I'll figure this out 😂

Yeah, I saw that in the description 😢. Sounds good! Reach out if you have any questions

**Pull request merged**